design-tokens/assets/ios/DesignTokens.xcassets/Brand.colorset/Contents.json

```
@@ -0,0 +1,17 @@
+{
+  "colors": [
+    {
+      "idiom": "universal",
+      "color": {
+        "color-space": "srgb",
+        "components": {
+          "red": "13",
+          "green": "189",
+          "blue": "139",
+          "alpha": "1"
```

Either of below works for us, but right now the alpha is set to 0.4%
```suggestion
          "alpha": "255"
```
```suggestion
          "alpha": "1.0"
```